src/components/section/ohri-form-section.component.tsx

```diff
@@ -47,12 +49,22 @@ const OHRIFormSection = ({ fields, onFieldChange }) => {
                 />
               );
               return isUnspecifiedSupported(fieldDescriptor) && fieldDescriptor.questionOptions.rendering != 'group' ? (
-                <div key={index}>
+                <div key={index} className={styles.parent}>
                   {qnFragment}
-                  <OHRIUnspecified question={fieldDescriptor} onChange={onFieldChange} handler={handler} />
+                  <div className={styles.tooltipWithUnspecified}>
+                    <OHRIUnspecified question={fieldDescriptor} onChange={onFieldChange} handler={handler} />
+                    {fieldDescriptor.questionInfo && <OHRITooltip field={fieldDescriptor} />}
+                  </div>
```

Comment:
same comment applies here about drying this out

Reply:
This has been resolved.